docs/development.md

```diff
@@ -0,0 +1,47 @@
+# Development
+
+## Running
+
+To run the app locally see [readme](../readme)
+
+## Running locally with Docker
+
+To run via a docker container:
+1. Perform the docker build with: 
+
+`docker-compose build app`
+2. You can optionally set build arguments by adding:
+
+`--build-arg arg_name='arg_value'`
+4. Run the container with:
+
+`docker-compose up app`
```

Comment:
💡 Preference 💡  I would add running as apart of the main README so that it all sits in one location. Add specific development information within this readme to clean it up but splitting running info could confuse

Reply:
yeah I think that's a good shout - I was kind of thinking linking running back to the main readme was a bit redundant. Will keep the config stuff in here and then move this stuff to the main